okhttp-coroutines/src/jvmMain/kotlin/okhttp3/JvmCallExtensions.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package okhttp3
+
+import kotlinx.coroutines.suspendCancellableCoroutine
+import okio.IOException
+import kotlin.coroutines.resumeWithException
+
+suspend fun Call.awaitResponse(): Response {
```

Comment:
Are `executeAsync` and `awaitResponse` synonyms? Why not just use `executeAsync` only?

Reply:
(I expect _await_ blocks waiting for something that’s already running, and doesn’t launch anything?)